extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -44,6 +47,8 @@ export class AmpAutoAds extends AMP.BaseElement {
       return;
     }
 
+    extensionsFor(this.win).loadElementClass(AD_TAG);
```

Comment:
As discussed offline, if we always insert amp-ad tag for amp-auto-ads, should we ask publisher to just put the script tag in `<head>` for sake of performance?

@dvoytenko